docs/source/install.rst

```diff
@@ -85,11 +93,12 @@ Build Dependencies
 
     conda create -n ucx -c conda-forge \
         automake make libtool pkg-config \
-        "python=3.12" setuptools "cython>=3.0.0"
+        "python=3.12" "setuptools>=64.0" "cython>=3.0.0"
 
 .. note::
-    The Python version must be explicitly specified here, UCX-Py currently supports
-    Python versions 3.10, 3.11, and 3.12.
+    The Python version must be explicitly specified here.
+    See https://docs.rapids.ai/install/ for the list of supported Python versions
+    for recent RAPIDS releases.
```

Comment:
I think this is not the most user-friendly way to phrase this here, unfortunately we do not have UCX-Py explicitly documented in https://docs.rapids.ai/install/ which has a different versioning scheme than the rest of RAPIDS, and thus it would be difficult for someone to understand what Python version is supported for UCX-Py based solely on the RAPIDS version. I agree the way it was documented isn't optimal and I can't think of a better way to do this differently than what it was either.

Reply:
Sure, that makes sense. My main motivation here was to avoid a hard-coded list of versions, since duplicating that kind of information in multiple places makes it more likely that it'll become out of date.

That said.... we do have a `rapids-reviser` template set up to replace this line when we roll out add-a-new-Python-version updates to all of RAPIDS.

I'll revert this change based on your comments here.

Reply:
I agree that would be ideal, unfortunately I think it's not helpful for the users given the different branching.

How does the `rapids-reviser` work? Can we easily make it replace arbitrary code/docs lines?

Reply:
Alright, no prob! I've reverted that change here.

> Can we easily make it replace arbitrary code/docs lines?

Yes totally! You give it a shell script and say "run this on these repos". This type of case is easy to automatically update there with `sed`, not a problem.